lib/pbench/test/unit/server/test_cache_manager.py

```diff
@@ -690,8 +729,35 @@ def test_cache_map_bad_dir_path(
                 "dir_name/subdir1/subdir14/subdir141/f1412_sym",
                 "dir_name/subdir1/subdir14/subdir141/f1412_sym",
                 "f1412_sym",
-                Path("dir_name/subdir1/f11.txt"),
-                CacheType.FILE,
+                Path("subdir1/f11.txt"),
+                CacheType.SYMLINK,
+                None,
+                CacheType.SYMLINK,
+            ),
+            (
+                "dir_name/subdir1/subdir14/subdir141/f1413_sym",
+                "dir_name/subdir1/subdir14/subdir141/f1413_sym",
+                "f1413_sym",
+                Path("subdir141"),
+                CacheType.SYMLINK,
+                None,
+                CacheType.SYMLINK,
+            ),
+            (
+                "dir_name/subdir1/subdir14/subdir141/f1414_sym",
+                "dir_name/subdir1/subdir14/subdir141/f1414_sym",
+                "f1414_sym",
+                Path("subdir141/f1411.txt"),
+                CacheType.SYMLINK,
+                None,
+                CacheType.SYMLINK,
+            ),
```

Comment:
Once you correct the symlink contents for these two, their `resolve_path` values will be relative to the root of the result area, and their `resolve_type` values will be `CacheType.DIRECTORY` and `CacheType.FILE` respectively.